hermes/probe/create_file_test.go

```diff
@@ -1 +1,201 @@
 package probe
+
+import (
+	"fmt"
+	"testing"
+	"bytes"
+	"context"
+
+	"cloud.google.com/go/storage"
+	"github.com/googleapis/google-cloud-go-testing/storage/stiface"
+)
+
+func TestFileName(t *testing.T) {
+	testUnits := [6]RandomHermesFile{ 
```

Comment:
Rather than writing this multiple times, it's normal to do something like 
```
tests := []struct{file RandomHermesFile
                            wantName string
                             wantError error}{
                              {RandomHermesFile{51, 12}, "",
                               "The file ID provided 51 wasn't in the required range [1, 50]"}
                              }
for _, test := range tests {
  got, err := test.file.FileName()
  if got != test.wantName {
    t.Errorf("descriptive error message: HermesFile{%v}.FileName(): wanted: %v got: %v", test.file, test.wantName, test.wantError)
}
<test for the error message goes here>
```

This makes the tests an awful lot easier to read.

Apologies for typos in my code example, github comment boxes are not a great editor :-P

This, here & for other tests.

Reply:
Changed! Wayyyyy more readable now thank you!